apps/web/src/lang/fr.json

```diff
@@ -2204,7 +2204,7 @@
     "description": "description of 'skill selection' section of the 'assessment details' dialog"
   },
   "ARSDO1": {
-    "defaultMessage": "Localisation, langue et sécurité.",
+    "defaultMessage": "Lieu, langue et sécurité.",
```

Comment:
This string doesn't include punctuation in English, nor should it in French.


## Screenshot

![Screen Shot 2023-12-11 at 14 01 41](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/98870d50-94f4-45f6-98ff-d1764ad6bf7d)

```suggestion
    "defaultMessage": "Lieu, langue et sécurité",
```